### PR TITLE
Diff command and merge --no-tags

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/lib/storm.py [hunk 1]
```diff
@@ -2583,6 +2583,25 @@ async def execStormCmd(self, runt, genr):
...
 
+class DiffCmd(Cmd):
+    '''
+    Generate a list of nodes with changes in the top layer of the current view.
+    '''
+    name = 'diff'
+
+    async def execStormCmd(self, runt, genr):
+
+        if runt.snap.view.parent is None:
```
[the remark this answers is not in the document]
Comment (invisig0th) [reply synapse/lib/storm.py:2594]: yep! it's similar to the merge use case where it doesn't make sense to diff data in layers that dont inherit.

File: synapse/lib/storm.py [hunk 2]
```diff
@@ -2649,16 +2671,20 @@ async def sync():
 
                 if adds:
                     addedits = [(node.buid, node.form.name, adds)]
-                    await runt.snap.view.layers[1].storNodeEdits(addedits, meta=meta)
+                    await runt.snap.view.parent.storNodeEdits(addedits, meta=meta)
```
[the remark this answers is not in the document]
Comment (invisig0th) [reply synapse/lib/storm.py:2674]: Already built that way ( checks for view.parent in the top of this method ) but yep!

[the remark this answers is not in the document]
Comment (rakuy0) [reply synapse/lib/storm.py:2674]: Ah. Missed that. Sweet :+1:

File: synapse/lib/storm.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rakuy0) [reply synapse/lib/storm.py:2594]: :+1:

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**